frb/surveys/catalog_utils.py

```diff
@@ -388,7 +389,85 @@ def convert_mags_to_flux(photometry_table, fluxunits='mJy'):
         fluxtable[mag][uplimit] = fluxtable[mag][uplimit] 
 
     return fluxtable
+
+def remove_duplicates(tab:Table, idcol:str)->Table:
+    """
+    In an astropy table if there are duplicate
+    entries, remove the duplicates.
+
+    Args:
+        tab (Table): A table of entries.
+        idcol (str): A column name that has unique ids
+            for each table entry.
+    Returns:
+        unique_tab (Table): A table with only the unique ids.
+    """
+    assert isinstance(tab, Table), "Please provide an astropy table."
+    assert isinstance(idcol, str), "Please provide a valid column name."
+    assert idcol in tab.colnames, "{} not a column in the given table".format(idcol)
+    # Sort entries first.
+    tab.sort(idcol)
+    # Get the duplicates.
+    duplicate_ids = np.where(tab[1:][idcol]==tab[:-1][idcol])[0]+1
+    tab.remove_rows(duplicate_ids)
+    return tab
+    
+def xmatch_and_merge_cats(tab1:Table, tab2:Table, tol:units.Quantity=1*units.arcsec,
+                        table_names:tuple=('1','2'), **kwargs)->Table:
+    """
+    Given two source catalogs, cross-match and merge them. This function 
```

Comment:
Maybe comment the Tables need to have `ra` and `dec` columns.

Reply:
Done.